lib/edge/render.js

```diff
@@ -3,11 +3,15 @@ import path from 'path'
 import * as edge from 'edge.js'
 import VerbatimTag from './VerbatimTag'
 
+const viewData = {
+    tabsEnabled: process.env.NODE_ENV !== 'production',
```

Comment:
```suggestion
    devOnly: process.env.NODE_ENV !== 'production',
```

Reply:
👍 good shout on having a more generic name, `isDev` could also work as the variable name

Reply:
I'm fine with either. I'll update #136 with whichever you go with.